pkgs/top-level/all-packages.nix

```diff
@@ -4442,7 +4442,7 @@ with pkgs;
 
   espanso = callPackage ../applications/office/espanso {
     inherit (darwin.apple_sdk.frameworks) AppKit Cocoa Foundation;
-    openssl = openssl_1_1;
+    # openssl = openssl_1_1;
```

Comment:
Just remove the line, no need to comment it out.

Reply:
Was commenting out for testing, will amend the commit.